src/documents/css/idg.css.styl

```diff
@@ -434,9 +434,78 @@ setActiveCategory(inputColor) {
     list-style-type: none;
 }
 
+/* Footer logos */
+
+footer svg {
```

Comment:
You can change this to "footer-logo-links svg" since it's the same as "footer svg". This way it's consistent with how we address the other styles.